ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/packages/index.tsx

```diff
@@ -99,24 +99,46 @@ const renderSubComponent = ({
 
   return (
     <div className='flex flex-col gap-4'>
-      {pkg.isModified && (
-        <div>
-          <Tooltip>
-            <TooltipTrigger>
-              <Badge
-                className={`border ${getIssueSeverityBackgroundColor('HINT')}`}
-              >
-                MODIFIED
-              </Badge>
-            </TooltipTrigger>
-            <TooltipContent>
-              The source code of the package has been modified compared to the
-              original source code, e.g., in case of a fork of an upstream Open
-              Source project.
-            </TooltipContent>
-          </Tooltip>
-        </div>
-      )}
+      {pkg.isModified ||
+        (pkg.isMetadataOnly && (
+          <div className='flex gap-2'>
+            <div className='font-semibold'>Flags:</div>
+            {pkg.isModified && (
+              <div>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <Badge
+                      className={`border ${getIssueSeverityBackgroundColor('WARNING')}`}
+                    >
+                      MODIFIED
+                    </Badge>
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    The package has been modified compared to the original
+                    package, e.g., in case of a fork of an upstream Open Source
+                    project.
+                  </TooltipContent>
+                </Tooltip>
+              </div>
+            )}
+            {pkg.isMetadataOnly && (
+              <div>
+                <Tooltip>
+                  <TooltipTrigger>
+                    <Badge
+                      className={`border ${getIssueSeverityBackgroundColor('WARNING')}`}
+                    >
+                      METADATA ONLY
+                    </Badge>
+                  </TooltipTrigger>
+                  <TooltipContent>
+                    This is a metadata only package.
```

Comment:
Maybe a bit more verbose (and "metadata-only" with a hypthen): "This is a metadata-only package that has no source code associated to it.".